### ISSUE TITLE
model_dependence.thermal_expansivity?

### ISSUE THREAD

**Issue by bobmyhill** [geodynamics/aspect#2516]
Do the material models require the option to set NonLinearDependency for thermal_expansivity? I suppose it would be very infrequently needed...

**Comment by gassmoeller** [comment]
Yes, and also this structure was never used in the way it was intended (in fact it is not used at all at the moment), so while in theory reasonable, it is currently just not worth the effort.
